### PR TITLE
Updating Lab1 for SC22.

### PR DESCRIPTION
**Pull request by t-brown**
Changes include
- pcluster version 3.2.0
- WRF version 4.4

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md [hunk 1]
```diff
@@ -24,18 +24,17 @@ These are used to run the WRF executable (wrf.exe) to simulate atmospheric event
 The model domain includes the entire Continental United States (CONUS), using 12-km grid spacing, which means that each grid point is 12x12 km.
 The full domain contains 425 x 300 grid points. After running the WRF model, post-processing will allow visualization of atmospheric variables available in the output (e.g., temperature, wind speed, pressure). 
 
-On the HPC Cluster, download the CONUS 12km test case from the [NCAR/MMM website](https://www2.mmm.ucar.edu/wrf/users/) into the **/shared** directory.
+On the HPC Cluster, download the CONUS 12km test case from the [tutorial bucket](https://sc22-hpc-labs.s3.amazonaws.com/wrf_simulation_CONUS12km.tar.gz) into the **/shared** directory.
```
Comment (mhuguesaws) [inline content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md:27]: The intent of the sentence was to say the origin of the test case, not from where it is downloaded from.

```suggestion
The CONUS 12km is a test case provided by NCAR and can be retrieve from the [NCAR/MMM website](https://www2.mmm.ucar.edu/wrf/users/). For this tutorial and convenience, you will download the CONUS 12km test from [Amazon S3](https://aws.amazon.com/s3/) bucket into the **/shared** directory of the HPC Cluster
```

File: content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md [hunk 2]
```diff
@@ -45,6 +44,7 @@ A copy of the WRF source code is part of the AMI and located  in __/opt/wrf-omp/
 cd /shared/conus_12km
 
 cp /opt/wrf-omp/src/run/{\
+CAMtr_volume_mixing_ratio,\
```
Comment (mhuguesaws) [inline content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md:47]: The test case was running without that file. What change ?

**Review by mhuguesaws — COMMENTED** [review]
let some comments. It is likely that the new images will be appear too large on the website. Please check

File: content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md (hunk 2 above)
Comment (t-brown) [reply content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md:47]: Version of WRF, `>= v4.4` needs this static file.

**Comment by t-brown** [comment]
>  It is likely that the new images will be appear too large on the website. Please check


I checked and they are OK, however I do agree they are huge, will grab smaller ones.

File: content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md [hunk 3]
```diff
@@ -24,18 +24,17 @@ These are used to run the WRF executable (wrf.exe) to simulate atmospheric event
 The model domain includes the entire Continental United States (CONUS), using 12-km grid spacing, which means that each grid point is 12x12 km.
 The full domain contains 425 x 300 grid points. After running the WRF model, post-processing will allow visualization of atmospheric variables available in the output (e.g., temperature, wind speed, pressure). 
 
-On the HPC Cluster, download the CONUS 12km test case from the [NCAR/MMM website](https://www2.mmm.ucar.edu/wrf/users/) into the **/shared** directory.
+The CONUS 12km is a test case provided by NCAR and can be retrieve from the [NCAR/MMM website](https://www2.mmm.ucar.edu/wrf/users/). For this tutorial and convenience, you will download the CONUS 12km test from [Amazon S3](https://aws.amazon.com/s3/) bucket into the **/shared** directory of the HPC Cluster
```
Comment (mhuguesaws) [inline content/03-hpc-aws-parallelcluster-workshop/11-conus-12km.md:27]:
```suggestion
The CONUS 12km is a test case provided by NCAR and can be retrieved from the [NCAR/MMM website](https://www2.mmm.ucar.edu/wrf/users/). For this tutorial and convenience, you will download the CONUS 12km test from [Amazon S3](https://aws.amazon.com/s3/) bucket into the **/shared** directory of the HPC Cluster
```

**Pull request merged**